test/test_other.py

```diff
@@ -5805,33 +5807,33 @@ def test_link_with_bad_o_in_a(self):
   def test_require(self):
     inname = test_file('hello_world.c')
     self.emcc(inname, args=['-sASSERTIONS=0'], output_filename='a.out.js')
-    output = self.run_process(config.NODE_JS + ['-e', 'require("./a.out.js")'], stdout=PIPE, stderr=PIPE)
-    assert output.stdout == 'hello, world!\n' and output.stderr == '', 'expected no output, got\n===\nSTDOUT\n%s\n===\nSTDERR\n%s\n===\n' % (output.stdout, output.stderr)
+    create_file('run.js', 'require("./a.out.js")')
+    output = self.run_js('run.js')
```

Comment:
```suggestion
    output = self.run_js('run.js', stderr=subprocess.STDOUT)
```
This is necessary to verify that stderr is empty, as the test did before? Or is that not really needed?

Reply:
Not necessary, but also because I use `assertEqual` on the line below this is implicitly checking that, because run_js combines the stderr and stdout of the node process by default.

Reply:
Hmm, `run_js` has `stderr=None` by default which means it isn't captured, doesn't it?

Reply:
Maybe you are looking at some other function.  For run_js `interleaved_output` is True by default and:

https://github.com/emscripten-core/emscripten/blob/57b21b8fdcbe3ebb523178b79465254668eab408/test/common.py#L872-L877

Reply:
Ah, you're right! I was looking here:

https://github.com/emscripten-core/emscripten/blob/57b21b8fdcbe3ebb523178b79465254668eab408/test/jsrun.py#L86-L87